test/unit/Insurance.js

```diff
@@ -35,7 +35,6 @@ const setup = deployments.createFixture(async () => {
         "TracerPerpetualSwaps",
         {
             libraries: {
-                Balances: libBalances.address,
```

Comment:
Doesn't the perp need LibBalances?

Reply:
It does, but @BenjaminPatch and I came to the (verbal) conclusion that you mustn't need to explicitly link `internal` libraries.